server/coordinator/procedure/ddl/common_util.go

```diff
@@ -67,18 +67,21 @@ func BuildCreateTableRequest(table storage.Table, shardVersionUpdate metadata.Sh
 	}
 }
 
-func GetShardVersionByTableName(clusterMetadata *metadata.ClusterMetadata, schemaName, tableName string, shardVersions map[storage.ShardID]uint64) (storage.Table, metadata.ShardVersionUpdate, error) {
+func GetTableMetadata(clusterMetadata *metadata.ClusterMetadata, schemaName, tableName string) (storage.Table, error) {
 	table, exists, err := clusterMetadata.GetTable(schemaName, tableName)
 	if err != nil {
-		return storage.Table{}, metadata.ShardVersionUpdate{}, err
+		return storage.Table{}, err
 	}
 	if !exists {
-		return storage.Table{}, metadata.ShardVersionUpdate{}, errors.WithMessage(procedure.ErrTableNotExists, "table not exists")
+		return storage.Table{}, errors.WithMessage(procedure.ErrTableNotExists, "table not exists")
 	}
+	return table, nil
+}
 
+func GetTableShardVersion(table storage.Table, clusterMetadata *metadata.ClusterMetadata, shardVersions map[storage.ShardID]uint64) (metadata.ShardVersionUpdate, error) {
```

Comment:
Maybe we should provide a better function name, how about `BuildShardVersionUpdate`?